experimental/packages/sdk-events/src/EventLoggerProvider.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as api from '@opentelemetry/api-events';
+import { EventLogger } from './EventLogger';
+import { LoggerProvider } from '@opentelemetry/sdk-logs';
+
+export class EventLoggerProvider implements api.EventLoggerProvider {
+  private _loggerProvider: LoggerProvider;
+
+  constructor(loggerProvider: LoggerProvider) {
+    this._loggerProvider = loggerProvider;
+  }
+
+  getEventLogger(
+    name: string,
+    version?: string | undefined,
```

Comment:
you don't need to add `undefined` if you're using `?` to mark as optional, since it will already be `undefined` if you don't pass anything.

Same for the `options?` below

Reply:
updated

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/4629/commits/b45cccb61f014f15701f2f1baf970a6ac5dd39da